docs/source/storage.rst

```diff
@@ -282,6 +282,8 @@ region references link to subsets of another Dataset. To identify region referen
 to the ``source`` and ``path``, the  py:class:`~hdmf_zarr.utils.ZarrReference` object will also need to
 store the definition of the ``region`` that is being referenced, e.g., a slice or list on point indices.
 
+In the future, the reference resolver classes will need to be updated to implement region references
```

Comment:
@mavaylon1  can you please move this line to the list of TODO items in the warning message right below it and add mention where the reference resolver classes are located.